api/endpoint-tests/activities/get.js

```diff
@@ -0,0 +1,41 @@
+const tap = require('tap'); // eslint-disable-line import/no-extraneous-dependencies
+const { getFullPath, login, request } = require('../utils');
+
+tap.test('activities endpoint | GET /activities', async getUsersTest => {
+
+  const url = getFullPath('/activities');
+
+  getUsersTest.test('when unauthenticated', async unauthenticatedTest => {
+    const { response, body } = await request.get(url);
+    unauthenticatedTest.equal(
+      response.statusCode,
+      403,
+      'gives a 403 status code'
+    );
+    unauthenticatedTest.notOk(body, 'does not send a body');
+  });
+
+  getUsersTest.test('when authenticated', async validTest => {
+    const cookies = await login();
+    const { response, body } = await request.get(url, {
+      jar: cookies,
+      json: true
+    });
+
+    body.sort((activity1, activity2) => ((activity1.id < activity2.id) ? -1 : 1));
```

Comment:
the sort here can be simplified to:

`body.sort((activity1, activity2) => (activity1.id - activity2.id))`

ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort

Reply:
Oh, good catch!  This also deals with the "they're the same" case, which I was ignoring.  I hate that this sort is even necessary but tap's `same` test expects array elements to be in the same order.  I guess that makes sense a lot of times, but for our purposes, I don't think we care about the order. 🤷‍♂️